docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1673,9 +1673,9 @@ Hive 3-related limitations
 
 * For security reasons, the ``sys`` system catalog is not accessible.
 
-* Hive's ``timestamp with local zone`` data type is not supported.
-  It is possible to read from a table with a column of this type, but the column
-  data is not accessible. Writing to such a table is not supported.
+* Hive's ``timestamp with local zone`` data type is mapped to
+  ``timestamp with time zone`` with UTC timezone. It only supports reading
```

Comment:
Why would writing not work? If the physical representation is the same, then writing should just work, I think. Have you tried it?

Reply:
It requires some additional changes, like adding cases to `isWritablePrimitiveType` and`ParquetSchemaConverter.getPrimitiveType`. We'll need support for writing eventually too - I can add it in this PR or in a following one.